app/assets/javascripts/discourse/widgets/search-menu.js.es6

```diff
@@ -97,6 +97,7 @@ export default createWidget('search-menu', {
     const results = [this.attach('search-term', { value: searchData.term, contextEnabled }),
                      this.attach('search-context', { contextEnabled })];
 
+    if (searchData.term)
```

Comment:
I didn't used braces because I don't want to touch line below for aligning. Anyway I will do it now.

No need for that. If length is `0` then `if (searchData.term)` will always return `false`